### PR TITLE
home work 05 done

### PR DESCRIPTION
**Pull request by verovkin**
home work 05 done

### CODE REVIEW

File: 05/task_2.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+Створити програму, яка буде друкувати в консоль “I love Python” кожні 4.2 секунди,
+поки її виконання не буде перервано вручну.
+"""
+
+import time
+
+while True:
+    print("I love Python")
+    time.sleep(2.4)
```
Comment (klymenok) [inline 05/task_2.py]: Almost right. The requirement is to sleep for 4.2 seconds

**Review by klymenok — CHANGES_REQUESTED** [review]
Small change is required

**Comment by verovkin** [comment]
corrected

**Review by klymenok — APPROVED** [review]
Good job!

**Pull request merged**